site/pt/beta/tutorials/keras/basic_text_classification.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Should this repetition occur?</p><blockquote>A primeira camada é uma camada&nbsp;<code>Embedding</code>&nbsp;layer (<code><em>Embedding</em></code><em>&nbsp;layer</em>).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>